addons/purchase/models/purchase.py

```diff
@@ -658,9 +658,18 @@ def retrieve_dashboard(self):
                 """
         self._cr.execute(query, (one_week_ago, self.env.company.id))
         res = self.env.cr.fetchone()
-        result['all_avg_order_value'] = round(res[0] or 0, res[3])
         result['all_avg_days_to_purchase'] = round(res[1] or 0, 2)
-        result['all_total_last_7_days'] = round(res[2] or 0, res[3])
+        
+        currency = self.env.company.currency_id
+        result['all_avg_order_value'] = formatLang(self.env, round(res[0] or 0, res[3]), currency_obj=currency)
+        result['all_total_last_7_days'] = formatLang(self.env, round(res[2] or 0, res[3]), currency_obj=currency)
+        
+        if currency.position == 'after':
```

Comment:
don't need it is already do in formatLang

Reply:
Yes. I changed it